guides/catalog/04 Introduction to Blob Storage.ipynb

```diff
@@ -169,11 +178,28 @@
    "outputs": [],
    "source": [
     "org = dl.auth.Auth().payload[\"org\"]\n",
-    "user_id = dl.auth.Auth().namespace\n",
-    "default_namespace = f\"{org}:{user_id}\"\n",
+    "user_hash = dl.auth.Auth().namespace\n",
+    "\n",
+    "try:\n",
```

Comment:
ditto from above - this code using and catching the assert is more confusing to look at then a simple if else.